### ISSUE TITLE
accessibility: radio and checkboxes should be within fieldsets

### ISSUE THREAD

**Issue by tahini** [chairemobilite/evolution#37]
And their labels should rather be the legend of the fieldset.  See https://www.w3.org/WAI/tutorials/forms/grouping/

Given what the Question.js file looks like now, it's not trivial at all to change. But oh joy! The Question.js class is ready to be moved to typescript!
